pallets/attestation/src/lib.rs

```diff
@@ -230,20 +240,21 @@ pub mod pallet {
 			delegation_id: Option<DelegationNodeIdOf<T>>,
 		) -> DispatchResult {
 			let source = <T as Config>::EnsureOrigin::ensure_origin(origin)?;
+			let payer = source.sender();
 			let attester = source.subject();
+			let deposit_amount = T::Deposit::get();
 
 			ensure!(
 				<ctype::Ctypes<T>>::contains_key(&ctype_hash),
 				ctype::Error::<T>::CTypeNotFound
 			);
-
 			ensure!(
 				!<Attestations<T>>::contains_key(&claim_hash),
 				Error::<T>::AlreadyAttested
 			);
 
 			// Check for validity of the delegation node if specified.
-			if let Some(delegation_id) = delegation_id {
+			let delegation_record = if let Some(delegation_id) = delegation_id {
 				let delegation = <delegation::DelegationNodes<T>>::get(delegation_id)
```

Comment:
C:
```suggestion
				let delegation = delegation::DelegationNodes::<T>::get(delegation_id)
```

Reply:
I would rather not touch lines that where not touched by this PR. If we want to change the style we do a separate PR.
I know you did this in your PRs. But having those things in a PR makes it hard to review.